pkg/server/updateResults.go

```diff
@@ -45,43 +48,87 @@ func updateResultRequest(ce *CreateExperiment) UpdateResults {
 	pm := kruize.NewProfileManager("")
 	for _, kubeobj := range ce.KubernetesObjects {
 		for _, contlist := range kubeobj.Containers {
-
-			// get queries from performanceProfile per container:
-			metricsList := pm.GetPerformanceProfileInstanceMetrics(ce.ClusterName, kubeobj.Namespace,
-				kubeobj.Name, contlist.ContainerName, ce.TrialSettings.MeasurementDuration)
-
 			//call function to parse the metrics:
 			windowSizeStr := strings.TrimSuffix(ce.TrialSettings.MeasurementDuration, "min")
 			windowSizeInt, _ := strconv.ParseInt(windowSizeStr, 10, 64)
-			startTime := time.Now()
```

Comment:
Is the startTime always going to be when the updateResults is posted? I wonder, would it be better to ask the user their desired start from computeRecommendations?

Reply:
@Anxhela21 Agree, we need to get the exact time user wants . We need to update our computeAPI for that. I added to our to do list.